packages/@ember/octane-app-blueprint/files/package.json

```diff
@@ -34,7 +34,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
-    "ember-data": "<%= emberData %>",
+    "ember-data": "^3.10.0",
```

Comment:
Instead of hard coding this to ^3.10.0, can it be the latest release? So that way as ember-data continues to release new versions, we don't need to keep updating the blueprint?

Reply:
That's actually the behavior we get from the `^` in `^3.10.0`, since `^` will resolve to the latest available 3.x version (but not less than 3.10)

Reply:
oh cool.